### ISSUE TITLE
AdaBoost settingsparameters don't have an effect

### ISSUE THREAD

**Issue by EKal-aa** [biolab/orange3#5384]
- [ ] **What's wrong?**

The settings/parameters in the AdaBoost widget don't have an effect. I used the little example in the documentation about AdaBoost (with iris dataset) and played around with the settings. But changes in the settings don't change the results of Test and score.

And the model cannot be saved with Save Model widget.

I tried different parameters:
number of estimators: 1 and 5
learning rate: 1 and 0.5
classification algorithm: SAMME.R (and one time SAMME)

The results of Test and Score are always the same.
Only different numbers for fixed seed changes the results, but for all variants in the same way.

![grafik](https://user-images.githubusercontent.com/55889795/113509987-b28a4980-9558-11eb-8f61-12ae06f8cde6.png)

![grafik](https://user-images.githubusercontent.com/55889795/113510015-cc2b9100-9558-11eb-86cb-d9c53f47aee0.png)

- [ ] **How can we reproduce the problem?** 
OWS-file:
[AdaBoost-Test1.zip](https://github.com/biolab/orange3/files/6254597/AdaBoost-Test1.zip)
- open the file with Orange-Canvas
- open Test and score and the different AdaBoost widgets

- [ ] **What's your environment?**

- Operating system: Win10 Pro 1803 and Win10 Enterprise LTSC 1809
- Orange version: 3.28.0 (on both computers)
- Python 3.8.5 and 3.7.4
- How you installed Orange: with the Orange3-3.28.0-Miniconda-x86_64.exe (64 bit) from Orange website.

**Comment by janezd** [comment]
Thanks for the report. But the widget works. :)

Try it on a more complex, larger data set, for instance Adult. On smaller data sets, the algorithm reaches an error of 0 in the first round and exits. (This behaviour is due to scikit-learn: Orange correctly passes all arguments to it.)

I tried saving and it works, too. Make sure that you give data to the widget, that is Datasets -> Ada Boost -> Save Model.
